plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/frontend.ts

```diff
@@ -285,6 +267,46 @@ const { actions, state } = store< VariableProductAddToCartWithOptionsStore >(
 				const matchedVariationId = matchedVariation?.variation_id;
 				productDataActions.setVariationId( matchedVariationId ?? null );
 			},
+			validateVariation() {
+
+				const { state: testState, actions: addToCartWithOptionsActions } = store<AddToCartWithOptionsStore>(
```

Comment:
I guess we don't need the state, here:

```suggestion
				const { actions: addToCartWithOptionsActions } = store<AddToCartWithOptionsStore>(
```

Also, won't `addToCartWithOptionsActions` contain the same as `actions`?

Reply:
I suppose it _is_ the same `actions` since you are currently extending the store vs having a new store name for the variable product parts of the stores.